### DISCUSSION TITLE
Troubleshooting Search Plugin Error on Website

### DISCUSSION THREAD

**Discussion by yanglbme** [squidfunk/mkdocs-material#7632] [Q&A] [answered]
Hello, I would like to ask a question.

Our website(https://doocs.github.io/leetcode/en/lc/1/) contains many pages, and we have enabled the search plugin. 

However, we often encounter the following error: `index.ts:113 GET https://doocs.github.io/leetcode/en/search/search_index.json net::ERR_FAILED 200 (OK)`, which prevents the entire website from loading the left sidebar properly. How can we resolve this issue?

![image](https://github.com/user-attachments/assets/38e827bb-138c-48d6-b5fc-5954949dfe6d)

**Comment by squidfunk** [comment, marked as answer]
I don't think this is a problem on our side. [Googling led me to](https://github.com/Alex313031/thorium/issues/306#issuecomment-2418020928):

> The error "net::ERR_FAILED 200 (OK)" appears when the size of the "blob_storage" folder in your browser profile reaches a certain limit (which depends on the size of free disk space).

The download succeeds, and I can browse your docs successfully and use the search, albeit it is slow, since the index is 4 MB in size. Note that performance and usability improvements in search are just around the corner, tracked in #6307.

**Comment by yanglbme** [comment]
Thank you very much for your reply. 

Recently, I noticed that when the system disk space is less than 4GB, the issue with my website occurs. Clearing up disk space can prevent this issue from happening.
